components/activity-section.jsx

```diff
@@ -0,0 +1,41 @@
+var React = require('react');
+var Illustration = require('../components/illustration.jsx');
+
+var ActivitySection = React.createClass({
+  propTypes: {
+    title: React.PropTypes.string.isRequired,
+    activities: React.PropTypes.array.isRequired
+  },
+  render: function () {
+    return (
+      <section className="activity-section">
+        <h2>
+          Section {this.props.title}
+        </h2>
+        {this.props.activities.map(function (activity, i) {
+          return (
+            <div className="activity" key={i}>
+              <Illustration height={122} width={122}
+              src1x={activity.image1x}
+              src2x={activity.image2x}
+              alt=""
+              link={activity.link}>
+                <div className="curriculum-description">
+                  <a href={activity.link}>
+                    <h3>{activity.title}</h3>
+                  </a>
+                  <p>
+                    <em>{activity.subtitle}</em>
+                  </p>
+                  <p className="description" dangerouslySetInnerHTML={{__html: activity.description}}></p>
```

Comment:
Um I'm gonna leave this unchanged for now @toolness :hushed: ... since now I don't know much about our localization plan/setup (technical-wise).  Maybe we can revisit this matter later when I know more about it? :eyes: